torch/_decomp/decompositions.py

```diff
@@ -1730,3 +1730,70 @@ def nll_loss_forward(
             result = result.sum() / total_weight
 
     return result, total_weight
+
+
+@register_decomposition(aten.mv)
+def mv(self, vec):
+    utils.check(
+        self.dim() == 2 and vec.dim() == 1,
+        lambda: f"matrix @ vector expected, got {self.dim()}, {vec.dim()}",
+    )
+    utils.check(
+        self.size(1) == vec.size(0),
+        lambda: f"size mismatch, got {self.size(0)}x{self.size(1)},{vec.size(0)}",
+    )
+    return (self * vec).sum(dim=1)
+
+
+@register_decomposition(aten.dot, disable_meta=True)
+def dot(self, other):
+    if self.is_complex():
```

Comment:
this is perf optimization for eager, so I don't think we need it in decomp?

Reply:
I'm just gonna keep it for compatibility's sake.